src/main/resources/rules/crime/means/decision/decision.drl

```diff
@@ -3,32 +3,52 @@ package rules.crime.means.decision
 import com.laa.model.crime.CrimeMeansDecisionReport;
 import com.laa.model.crime.CourtType;
 import com.laa.model.crime.CrimeAssessmentResult;
-
-
 import java.math.BigDecimal;
 
 
 rule "adjusted income limit check"
 	when
-		$decision : CrimeMeansDecisionReport( adjustedIncome !=null && adjustedIncome.compareTo(new BigDecimal("12475")) < 0)
+		$decision : CrimeMeansDecisionReport( adjustedIncome != null && adjustedIncome.compareTo(new BigDecimal("12475")) < 0)
 	then
 		$decision.setAdjustedIncomeBelowLowerThreshold(true);
 		update($decision);
 		
 end
 
 
-rule "Verify if means assessment is passed for megistrate court case when adjustedIncomeBelowLowerThreshold is true"
```

Comment:
spelling error magistrate, also rename the rule as it now only looks at income not the court

Reply:
done